NitroxServer/Serialization/World/WorldData.cs

```diff
@@ -1,12 +1,12 @@
 ﻿using System;
 using System.Collections.Generic;
-using NitroxModel.DataStructures;
 using NitroxServer.GameLogic;
 using NitroxServer.GameLogic.Bases;
 using NitroxServer.GameLogic.Entities;
 using NitroxServer.GameLogic.Items;
 using NitroxServer.GameLogic.Vehicles;
 using ProtoBufNet;
+using DTO = NitroxModel.DataStructures;
```

Comment:
Can NitroxServer files directly rely on datastructures as before?  I don't see a reason here to explicitly call out DTO.  Also, we may be able to completely decouple server from unity (outside of resource parsing).

Reply:
Fair enough to me. Decoupling the server is indeed the end goal. Not sure I reach it with this PR only though.